packages/paste-website/src/pages/components/topbar/index.mdx

```diff
@@ -123,11 +359,45 @@ User-specific controls (aligned to the right) such as:
     ProductSegmentIcon,
     ProductFlexIcon,
     ProductEmailAPIIcon,
+    ChevronDownIcon,
     SearchIcon,
+    StatusMenu,
```

Comment:
praise: wow, that's one heck of a composition

Reply:
seriously